lib/query.js

```diff
@@ -21,13 +21,23 @@ const idMatch = _.curry((id, obj) => id + '' === obj.id + '');
 
 const dateIsUnix = _.compose(_.equals(0), _.modulo(_.__, 1));
 
+const dateIsMs = _.compose(
```

Comment:
All this Ramda is unnecessary
```js
const dateIsMs = (num) => (parseInt(num) > LAGEST_UNIX_TIMESTAMP)
```

Reply:
but... the ramda!

Will update, thanks!